easybuild/easyconfigs/r/R/R-4.0.0-foss-2020a.eb

```diff
@@ -2125,15 +2125,14 @@ exts_list = [
         'checksums': ['3a1f166f1c71b5877a2acca1384ec6c9b430b67af67ef26125f2abbb53c66206'],
     }),
     ('liquidSVM', '1.2.4', {
-        'checksums': ['15a9c7f2930e2ed3f4c5bcd9b042884ea580d2b2e52e1c68041600c196046aba'],
         'patches': ['liquidSVM-1.2.4-fix_ppc_and_aarch64_build.patch'],
+        'preinstallopts': 'LIQUIDSVM_TARGET="empty"',
         'checksums': [
-            '15a9c7f2930e2ed3f4c5bcd9b042884ea580d2b2e52e1c68041600c196046aba',
+            '15a9c7f2930e2ed3f4c5bcd9b042884ea580d2b2e52e1c68041600c196046aba',  # liquidSVM_1.2.4.tar.gz
             # liquidSVM-1.2.4-fix_ppc_and_aarch64_build.patch
             '46b09e441c3b59af535f20d8db0dee7f1d6a7ddd511175d252115b53cb8b86f8',
         ],
         # Don't add optimization flags by liquidSVM which may not be known e.g. on PPC
```

Comment:
@terjekv This should go right above the `'preinstallopts': 'LIQUIDSVM_TARGET="empty"',` line (so maybe move that down again)?

Reply:
Ah, hm, right. Sorted that now. Not *quite* sure what I was thinking at the time.